waltz-tools/src/main/java/com/wepay/waltz/tools/client/ClientCli.java

```diff
@@ -167,7 +167,10 @@ protected void processCmd(CommandLine cmd) throws SubCommandFailedException {
                 if (avgInterval < 0) {
                     throw new IllegalArgumentException("Found negative: interval must be greater or equals to 0");
                 }
-                String configFilePath = cmd.getOptionValue("cli-config-path");
+                String configFilePath = cmd.getOptionValue(
+                        "cli-config-path",
+                        System.getProperty("waltz.config", "/etc/waltz/waltz.cfg")
```

Comment:
maybe `/etc/waltz/waltz-client.cfg`. I don't want this to be confused with a server config or a storage config.
Same for other CLI tools.

Reply:
I'm not sure I understand how the configs are different.  In our deployments, I am currently using the same config file for all of these things.